core/crypto/parser.go

```diff
@@ -86,6 +87,8 @@ func replaceWithGenerated(s *string, expression string, ranges [][]byte,
 			alphabet += everything
 		case `\d`:
 			alphabet += numbers
+		case `\s`:
```

Comment:
`\d` is regex for "decimal"

`\s` is regex for "whitespace"; so it might be confusing for the end user.

instead of `[\s]` it would be better to let users be explicit and use something like `[?!;,:.]`.

Reply:
Agree, something like `[?!;,:.]` works fine with limit special characters, I changed `\s` to `\symbol` to avoid confusing.